web/cypress/fixtures/export-route-dataset.json

```diff
@@ -8,8 +8,8 @@
     "openapi": "3.0.0",
     "paths": {
       "/{params}": {
-        "get": {
-          "operationId": "route_name_0GET",
+        "post": {
```

Comment:
why change `get` to `post`?

Reply:
The test case for import and export is using POST, then I updated the dataset file.